javascript-data-model/order-history.js

```diff
@@ -0,0 +1,46 @@
+var orderHistory = [
+  {
+    orderedDate: 'August 4, 2020',
+    totalPrice: '$34.00',
+    shipTo: 'JS Masher',
+    orderNum: '114-3941689-8772232',
+    deliveredDate: 'Aug 8, 2020',
+    orderTitle: ['JavaScript for impatient programmers'],
+    orderAuthor: ['Rauschmayer, Dr. Axel'],
+    returnWindow: 'Sep 7, 2020',
+    price: ['$31.55']
+  },
+  {
+    orderedDate: 'July 19, 2020',
+    totalPrice: '$44.53',
+    shipTo: 'JS Masher',
+    orderNum: '113-9984268-1280257',
+    deliveredDate: 'Jul 20, 2020',
+    orderTitle: ['The Timeless Way of Building'],
```

Comment:
my recommendation for each of the orders is to create an array `items` which has an object for each item ordered in it so that it is easy to have many items from all different departments.